RubberduckBaseMetaProject.csproj

```diff
@@ -0,0 +1,71 @@
+<Project Sdk="Sunburst.NET.Sdk.WPF/1.0.47">
+  <!-- We're specifying the Sdk here to make appveyor happy, but this generates warning MSB4011 -->
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <Company>Rubberduck-VBA</Company>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <TargetFramework>net46</TargetFramework>
+    <LangVersion>7.2</LangVersion>
+    <SolutionDir>$(MSBuildProjectDirectory)</SolutionDir>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+    <RunCodeAnalysis>true</RunCodeAnalysis>
+    <RestorePackages>true</RestorePackages>
+    <!-- do not unify output paths by default -->
+    <UnifyOutputPath Condition=" '$(UnifyOutputPath)' == ''">false</UnifyOutputPath>
```

Comment:
If all projects that use `UnifyOutputPath` have moved to the Meta solution, that would be a good opportunity to remove that from the RubberduckBaseProject.cs and just default it to `true` here.

Reply:
`Rubberduck.Deployment` is still in the core solution and needs its path unified so I can't remove it from the base project, but I can do that as default for the meta projects.